src/foundry/common/utils/primitives.mjs.d.ts

```diff
@@ -92,19 +92,22 @@ declare interface Number {
 
   /**
    * Round a number to the nearest number which is a multiple of a given interval
-   * @param interval - The interval to round the number to the nearest multiple of
-   *                   (default: `1`)
+   * @param interval- The interval to round the number to the nearest multiple of
```

Comment:
There is a space missing and  the default value of `interval` is not documented

```suggestion
   * @param interval - The interval to round the number to the nearest multiple of
                       (default: `1`)   
```

Reply:
done